tools/hive-bigquery/init_script.py

```diff
@@ -0,0 +1,147 @@
+    optional.add_argument(
+        '--hive-server-port', required=False,
+        default=10000, type=int, help="Hive server port, defaults to 10000")
+    optional.add_argument(
+        '--hive-server-username', required=False,
+        default=None, help="Hive username, defaults to None")
+    required.add_argument('--hive-database', required=True,
+                          help="Hive database name")
+    required.add_argument('--hive-table', required=True, help="Hive table name")
+    required.add_argument('--project-id', required=True, help="GCP Project ID")
+    required.add_argument('--bq-dataset-id', required=True,
+                          help="Existing BigQuery dataset ID")
+    optional.add_argument('--bq-table', required=False, default='',
+                          help="BigQuery table name")
+    required.add_argument(
+        '--bq-table-write-mode', required=True,
+        choices=['overwrite', 'create', 'append'],
+        help="BigQuery table write mode.\nUse overwrite to overwrite the "
+             "previous runs of migration.\nUse create if you are migrating "
+             "for the first time.\nUse append to append to the existing "
+             "BigQuery table")
+    required.add_argument(
+        '--gcs-bucket-name', required=True,
+        help="Existing GCS Bucket name. Provide either "
+             "gs://BUCKET_NAME or BUCKET_NAME. Hive data will be copied to this"
+             "bucket and then loaded into BigQuery")
+    optional.add_argument(
+        '--incremental-col', required=False,
+        help="Provide the incremental column name if present in the Hive table")
+    optional.add_argument(
+        '--use-clustering', required=False,
+        default="False", choices=["False", "True"],
+        help="Boolean to indicate whether to use clustering in BigQuery if "
+             "supported")
+    required.add_argument(
+        '--tracking-database-host', required=False, default='localhost',
+        help="Cloud SQL Tracking database host address, defaults to localhost")
+    optional.add_argument(
+        '--tracking-database-port', required=False, default=3306,
+        help="Port which you used for running cloud sql proxy, defaults to 3306")
+    optional.add_argument(
+        '--tracking-database-user', required=False, default='root',
+        help="Cloud SQL Tracking database user name, defaults to root")
+    required.add_argument(
+        '--tracking-database-db-name', required=True,
+        help="Cloud SQL Tracking db name.Ensure you provide the same db name "
+             "if you have migrated previously")
+
+    return parser.parse_args()
+
+
+def initialize_variables():
+    """Initialize variables"""
+
+    args = arg_pars()
+
+    # Gets password from the user to coonect to Cloud SQL tracking database
+    tracking_database_password = getpass.getpass(
+        'Enter Cloud SQL tracking database password:')
+
+    time_format = datetime.datetime.now().strftime("%Y_%m_%d_%H_%M_%S")
```

Comment:
Style note, when we set globals like this should be all caps.

Reply:
The [style guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#316-naming) stated that for global constants, we need to use all caps, but nothing on function level constants. Should I proceed and change this to caps?

Reply:
Changing this to all caps results in not following the pylint convention.
`
C:104, 4: Variable name "TIME_FORMAT" doesn't conform to snake_case naming style (invalid-name)
`
pylint wants function locals to be in lowercase